dist/test_data.py

```diff
@@ -112,8 +112,8 @@ def __ge__(self, other):
 ]
 
 thread_count = [
-    Config('thread_count', 1, r'''
-        Specifies the number of threads that will be used to perform a certain function.''')
+    Config('thread_count', 0, r'''
```

Comment:
Here I'm just setting the default thread_count to 0, thus any test that doesn't define a thread count won't be doing things by accident.